ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/influxdb.j2

```diff
@@ -0,0 +1,52 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - influxdb #}
+{% if influxdb is arista.avd.defined %}
+
+## Influxdb Streaming
+
+### Influxdb Summary
+{%     if influxdb.source_group_standard_disabled is arista.avd.defined(true) %}
+
+Source Group Standard Disabled : {{ influxdb.source_group_standard_disabled }}
+{%     endif %}
+{%     if influxdb.destinations is arista.avd.defined %}
+
+#### Telemetry Outputs Summary
```

Comment:
Some headings say "InfluxDB". One say "InfluxDB streaming" and these say "Telemetry". I do not know the details and lingo used with InfluxDB, so I will just request you to confirm that these headings are correct. Also please remove the `summary` from the table headings.

Reply:
Maybe:
```suggestion
#### InfluxDB Telemetry Destinations
```

Reply:
Fixed.